platform-sdk/docs/proposals/template.md

```diff
@@ -0,0 +1,135 @@
+# Proposal Title Goes Here
+
+---
+
+## Summary
+
+Give a 1-3 sentence summary of the capability in the design proposal.
+
+| Metadata           | Entities                            | 
+|--------------------|-------------------------------------|
+| Designers          | John, Mary, alpha-team              |
+| Functional Impacts | Services, DevOps, Mirror Node, etc. |
+| Related Proposals  | Proposal-1, Proposal-2              |
+| HIPS               | HIP-1, HIP-2,                       |
+
+---
+
+## Purpose and Context
+
+Why is the design proposal necessary? Describe the background and purpose of the design proposal at a high level.
+
+### Dependencies, Interactions, and Implications
+
+What are the dependencies and pre-requisites of this proposal?  
+What other proposals, capabilities, and teams are impacted by this proposal?
+What does the proposal enable once it is completed?
+
+Remove this section if not applicable.
+
+### Requirements
+
+Describe the requirements and acceptance criteria that the design proposal must satisfy.
+
+### Design Decisions
+
+Describe the decisions made and the reasons why.
+
+#### Alternatives Considered
+
+Describe any alternatives considered and why they were not chosen.
+
+If possible, provide a table illustrating the options, evaluation criteria, and scores that factored into the decision.
+
+
+---
+
+## Changes
```

Comment:
The notion page we took notes on also included these items:

- Any new metrics or changes to existing metrics
- Changes to default configurations or addition of new configurations